site/src/sphinx/server-annotated-service.rst

```diff
@@ -239,11 +242,25 @@ annotation or ``Optional<?>`` class, e.g. ``hello2`` and ``hello3`` method in th
         }
     }
 
+If an HTTP POST request with a ``Content-Type: x-www-form-urlencoded`` is received and any `@Param`_ value
+is not shown in the path pattern, Armeria will aggregate the received request and decode its body as an
```

Comment:
- any `@Param` value is now shown -> no `@Param` value appears
- an url-encoded -> a URL-encoded